### PR TITLE
RS-272: Get the benchmark comparisons on the landing page

### PR DESCRIPTION
**Pull request by AnthonyCvn**
- benchmark comparisons next to the benefits
- new landing page title and subtitle

### CODE REVIEW

**Comment by AnthonyCvn** [comment]
@atimin I'd need the benchmark on your drive for InfluxDB + MinIO to update the table.

**Review by atimin — CHANGES_REQUESTED** [review]
I like the idea, but  maybe it is possible to optimize the page layout .

File: src/components/PerformanceComparison/index.tsx [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+  <table>
+    <thead>
+      <tr>
+        <th>Chunk Size</th>
+        <th>Operation</th>
+        <th>ReductStore, %</th>
+      </tr>
+    </thead>
+    <tbody>
+      {data.map((row, index) => (
```
[the remark this answers is not in the document]
Comment (AnthonyCvn) [reply src/components/PerformanceComparison/index.tsx]: @atimin I agree, let me know what you think about the new one

**Review by atimin — APPROVED** [review]
Cool!

**Pull request merged**